src/themis/secure_comparator_t.h

```diff
@@ -24,6 +24,16 @@
 
 typedef themis_status_t (*secure_compare_handler)(secure_comparator_t *comp_ctx, const void *input, size_t input_length, void *output, size_t *output_length);
 
+/*
+ * Common structure for both peers (Alice and Bob 
+ * as denoted in paper (https://eprint.iacr.org/2015/1180.pdf))
+ * 
+ * secret             -> value to compare of each peer (x - for Alice, y - for Bob)
+ * rand, rand2, rand3 -> random values of each peer(r, a2, a3 - for Alice; s, b2, b3 - for Bob)
+ * g2, g3, P, Q       -> intermediate parameters of each peer while protocol execution (G2a, G3a, Pa, Qa - for Alice; G2b, G3b, Pb, Qb - for Bob)
+ * Pp, Qa_Qb, g3p     -> temporable parameters
```

Comment:
I think this needs to be broken down:
`Pp, g3p` - intermediate parameters received from your peer
`Qa_Qb` - local temporary intermediate parameter